.babelrc

```diff
@@ -0,0 +1,17 @@
+{
+  "presets": [
+    ["env", {
+      "targets": {
+        "browsers": ["last 2 versions"]
+      }
+    }],
+    "stage-3"
+  ],
+  "plugins": [
+    "transform-runtime",
+    ["module-resolver", {
+      "root": ["./"]
```

Comment:
Note that in JS files this plugin allows for absolute imports, so no more `../../../` unmaintainability:
```javascript
import artworksApp from 'desktop/apps/artworks'
```

Reply:
Ah, yes of course! Game changer!

Reply:
If only it worked in coffeescript files :/